stdlib/3/http/client.pyi

```diff
@@ -135,6 +136,17 @@ else:
         def getcode(self) -> int: ...
         def begin(self) -> None: ...
 
+class HTTPConnectionProtocol(Protocol):
```

Comment:
Only one more comment: Could you add a comment that this is stubs-only and not available at runtime?